packages/react/src/useBatchingCallback.ts

```diff
@@ -31,7 +31,10 @@ import type { DependencyList } from 'react';
  * @param [deps] The optional callback dependencies.
  * @return The memoized batching function.
  */
-function useBatchingCallback<T extends (...args: unknown[]) => unknown>(
+function useBatchingCallback<
+  T extends (...args: [event: KeyboardEvent]) => void
```

Comment:
Not sure if that's the best solution here. LMK if you have any thoughts.

Reply:
Doesn't seem so, getting the following error then:
`Argument of type '(...args: unknown[]) => void' is not assignable to parameter of type 'T'.`